Dockerfile.release

```diff
@@ -10,11 +10,11 @@ RUN set -xe && \
     rm -r /root/.cache
 
 ARG ICLOUDPD_VERSION
-ADD icloudpd-latest-py2.py3-none-any.whl /tmp
```

Comment:
I'm not really sure what this .whl is for, so I can't really say something to that change. Why are the user additions commented out? AFAIR we had an issue about user rights (cron, suid, something) in relation with docker... Is docker on other platforms still fine? Can we test this in any unit test?

Reply:
I noticed this the other night too. I was going to try building, but couldn't find this file and presumed the Dockerfile build would not be able to find it.

Assumed this must exist on the machine that is building the Docker images.

Reply:
We have two dockerfile`s in the repo:
- `Dockerfile` that uses python base image and uses current source tree to build image (including testing, pulling all dependencies etc); it produces *fat* image (1.7G from latest master).
- `Dockerfile.release` uses alpine base and relies on the rest of release scripts (test, lint, python package into .whl) for all the work except packaging docker image; it produces 70k image and this is the one that is published to dockerhub.

This PR is about fixes to `Dockerfile.release`.

To produce .whl locally, you need to run `scripts\build`

Reply:
> I'm not really sure what this .whl is for, so I can't really say something to that change. Why are the user additions commented out? AFAIR we had an issue about user rights (cron, suid, something) in relation with docker... Is docker on other platforms still fine? Can we test this in any unit test?

Tested on Mac before and after the change -- both versions worked.
Windows broken before and worked after the change.
Linux -- did not test.

Testing docker automatically would be awesome:
1. need to figure out how to simulate icloud traffic, e.g. from existing vcr recordings.
1. need to figure out how GitHub actions work with docker
1. need to figure out how GitHub support different platforms and ideally architectures, so we can test Win/Mac/Lin and x64/arm

That is important, but separate enhancement to our dev lifecycle.

Reply:
Tested on Linux (Ubuntu 20.04) before and after the change -- both versions worked.